.ci/pipeline_definitions

```diff
@@ -1,24 +1,12 @@
 gardener-website-generator:
-  template: 'default'
   base_definition:
-    traits:
-      version: ~
-      scheduling:
-        suppress_parallel_execution: true
-    repo:
-      name: 'gardener_website_generator'
-      trigger: true
-      cfg_name: 'github_com'
```

Comment:
`github_com` and trigger are the default values anyways, so it was not needed in the first place :-)